1.8/koding/README.md

```diff
@@ -0,0 +1,58 @@
+# How to use Koding on DC/OS
+
+[Koding](https://wwww.koding.com/) is cloud development platform.
+
+- Estimated time for completion: 10 minutes
+- Target audience: Development teams
+- Scope: Learn how to use Koding on DC/OS.
+
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Marathon-LB](#install-marathon-lb)
+- [Install Koding](#install-koding)
+- [Access Koding](#access-koding)
+- [Use Koding in production](#use-koding-in-production)
+
+## Prerequisites
+
+- A running DC/OS 1.8 cluster with at least 1 [private agent](https://dcos.io/docs/1.8/overview/concepts/#public) node with 2 CPUs and 1 GB of RAM available.
```

Comment:
This differs from the statement below 'requires 2 CPUs and 8GB of RAM'. Suggest that you only state the requirements here and remove below.